libs/libcommon/README.md

```diff
@@ -16,7 +16,7 @@ Set the common environment variables to configure the following aspects:
 - `COMMON_HF_ENDPOINT`: URL of the HuggingFace Hub. Defaults to `https://huggingface.co`.
 - `COMMON_HF_TOKEN`: App Access Token (ask moonlanding administrators to get one, only the `read` role is required) to access the gated datasets. Defaults to empty.
 - `COMMON_LOG_LEVEL`: log level, among `DEBUG`, `INFO`, `WARNING`, `ERROR`, and `CRITICAL`. Defaults to `INFO`.
-
+-  `COMMON_CONTENT_MAX_SIZE`: the maximum size in bytes of the content to be stored in the cache. Defaults to `100_000_000`
```

Comment:
Could you move it to the "CACHE_" namespace? It would make more sense

Reply:
Will I have to move also that config to `CacheConfig`? If so, I am afraid to have the `post_init` `connect_to_cache_database` running.
Maybe add another namespace but will have to create another config class no handle it

Reply:
I'm currently working on removing all the "post_init" things from the config: we will have a new "Resource" object, that will be allocated and then released (database, log, storage, etc.)
So, if I finish soon enough, we could move this parameter to the cache config, it would be more logical.

Reply:
As per last commit, I kept  the logic in the worker